### PR TITLE
[Pipelines] Switch to OpenShift Pipelines cluster pool

### PR DESCRIPTION
**Pull request by divyansh42**
This PR updates cluster pools for the repos:
- `task-containers`
- `task-maven`
- `task-git`
- `task-openshift`

`task-buildpacks` update will be done later once we migrate the docker image from docker.io to other alternatives

### CODE REVIEW

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by deepsm007** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by 2uasimojo** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

File: ci-operator/config/openshift-pipelines/task-buildpacks/openshift-pipelines-task-buildpacks-main.yaml [hunk 1]
```diff
@@ -35,8 +35,8 @@ tests:
   cluster_claim:
```
Comment (deepsm007) [inline ci-operator/config/openshift-pipelines/task-buildpacks/openshift-pipelines-task-buildpacks-main.yaml:35]:
```suggestion
  cluster: build05
  cluster_claim:
```
And run `make jobs` to update the test point correct cluster

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-113-ocp-414-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-git-main-osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-git-main-osp-115-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-git-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-maven-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-maven-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-maven-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-containers-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-openshift-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-openshift-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-openshift-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-openshift-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-openshift-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-openshift-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-containers-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-containers-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-containers-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-openshift-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
ci/rehearse/openshift-pipelines/task-buildpacks/main/osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse ci/rehearse/openshift-pipelines/task-buildpacks/main/osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-115-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-openshift-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-buildpacks-main-osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-openshift-main-osp-115-ocp-415-e2e  pull-ci-openshift-pipelines-task-openshift-main-osp-nightly-ocp-415-e2e pull-ci-openshift-pipelines-task-openshift-main-osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-maven-main-osp-114-ocp-415-e2e pull-ci-openshift-pipelines-task-maven-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-maven-main-osp-nightly-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-git-main-osp-114-ocp-415-e2e

**Comment by divyansh42** [comment]
/pj-rehearse pull-ci-openshift-pipelines-task-git-main-osp-115-ocp-415-e2e pull-ci-openshift-pipelines-task-git-main-osp-114-ocp-415-e2e

**Comment by vdemeester** [comment]
``` error: resource mapping not found for name: "git-clone" namespace: "" from "STDIN": no matches for kind "StepAction" in version "tekton.dev/v1beta1" ```

Those errors on `task-git` can be fixed in the repo 👼🏼

**Review by vdemeester — APPROVED** [review]

**Pull request merged**